prisma/migrations/20240320184319_update_organization/migration.sql

```diff
@@ -0,0 +1,24 @@
+-- CreateEnum
+CREATE TYPE "OwnershipType" AS ENUM ('PRIVATE', 'GOVERNMENT');
+
+-- AlterTable
+ALTER TABLE "organization" ADD COLUMN     "isInclusiveSpace" BOOLEAN,
+ADD COLUMN     "ownershipType" "OwnershipType";
+
+-- CreateTable
+CREATE TABLE "_OrganizationToSpecialization" (
+    "A" UUID NOT NULL,
+    "B" UUID NOT NULL
+);
```

Comment:
@alex-popov-tech , is it some convention for naming here? It might confuse, so I'd recommend renaming:

A -> organiationId
B -> specializationId
_OrganizationToSpecialization -> OrganizationSpecializations

Reply:
kinda yes, its default names generated by Prisma when we add m2m relation to schema it auto-creates table by pattern `_${capitalize(firstModelName)}To${capitalize(secondModelName)}` and with columns `A` for first model id and `B` for second one :) 

So to change these we would need to add m2m model in schema manually, which kinda looks redundant as we don't have any more fileds in relation table

Reply:
That's weird, but OK if that's convention.